### PR TITLE
Show multiple vision targets simultaneously

### PR DESCRIPTION
**Pull request by having11**
https://github.com/SubZero-Robotics/2024-CommandRobot/assets/16725165/d26e3476-e27d-4296-b711-5bde96609f42

### CODE REVIEW

File: src/main/cpp/utils/TargetTracker.cpp [hunk 1]
```diff
@@ -11,9 +11,35 @@
...
-    : m_config{config}, m_intake{intake}, m_scoring{scoring}, m_drive{drive} {}
+    : m_config{config}, m_intake{intake}, m_scoring{scoring}, m_drive{drive} {
+  m_trackedTargets = std::vector<TrackedTarget>(m_config.maxTrackedItems);
+}
 
 std::vector<DetectedObject> TargetTracker::GetTargets() {
+  if (!frc::RobotBase::IsReal()) {
+    static int counter = 0;
+    return {
+        DetectedObject(1, 0.75,
```
Comment (nobody5050) [inline src/main/cpp/utils/TargetTracker.cpp:22]: Now that we're doing more "sim only" stuff like this you should have constants for these (maybe a namespace?) (maybe unit tests using them?)

File: src/main/cpp/utils/TargetTracker.cpp [hunk 2]
```diff
@@ -120,7 +171,7 @@ units::inch_t TargetTracker::GetDistanceToTarget(const DetectedObject& target) {
   frc::SmartDashboard::PutNumber("TargetTracker Pixel Width", pixelWidth);
 
   auto otherDistance =
-      pixelWidth > 0
+      pixelWidth > 3
```
Comment (nobody5050) [inline src/main/cpp/utils/TargetTracker.cpp:174]: magic number?

**Review by nobody5050 — CHANGES_REQUESTED** [review]

File: src/main/cpp/utils/TargetTracker.cpp (hunk 2 above)
Comment (having11) [reply src/main/cpp/utils/TargetTracker.cpp:174]: yes

File: src/main/cpp/utils/TargetTracker.cpp (hunk 1 above)
Comment (having11) [reply src/main/cpp/utils/TargetTracker.cpp:22]: agreed

Comment (having11) [reply src/main/cpp/utils/TargetTracker.cpp:22]: Let's do this in the cleanup branch this week

**Review by nobody5050 — APPROVED** [review]

**Pull request merged**